src/qt/intro.cpp

```diff
@@ -381,6 +382,9 @@ void Intro::UpdatePruneLabels(bool prune_checked)
     }
     ui->lblExplanation3->setVisible(prune_checked);
     ui->pruneGB->setEnabled(prune_checked);
+    static const uint64_t nPowTargetSpacing = 10 * 60;  // from chainparams, which we don't have at this stage
+    static const uint32_t expected_block_data_size = 2250000;  // includes undo data
```

Comment:
7d40a4805229618a364dc53a84cd5af2ac273b7d, nit:
```suggestion
    static constexpr uint64_t nPowTargetSpacing = 10 * 60;  // from chainparams, which we don't have at this stage
    static constexpr uint32_t expected_block_data_size = 2250000;  // includes undo data
```